### PR TITLE
Date input: input classname fix

### PR DESCRIPTION
**Pull request by ChaitanyaGadodia**

### CODE REVIEW

File: packages/pebble-web/src/components/DateInput.tsx [hunk 1]
```diff
@@ -107,7 +108,7 @@ export default class DateInput extends React.PureComponent<
                   ...(inputProps && inputProps.inputProps),
                   onChange
                 }}
-                className={inputStyle}
+                className={cx(inputStyle, inputProps && inputProps.className)}
```
Comment (zhirzh) [inline packages/pebble-web/src/components/DateInput.tsx:111]: `DateInputProps` has no `className` field

https://github.com/anarock/pebble/blob/f1409fcd3a2a35d57988f0f519c4c0e6fe6c32ef/packages/pebble-web/src/components/typings/DateInput.ts#L5-L11

Comment (ChaitanyaGadodia) [reply packages/pebble-web/src/components/DateInput.tsx:111]: so?

Comment (zhirzh) [reply packages/pebble-web/src/components/DateInput.tsx:111]: - `inputProps` in expression `inputProps && inputProps.className` is of type `DateInputProps`
- `DateInputProps` has no `className` field
- so `inputProps.className` should give TS error

Comment (ChaitanyaGadodia) [reply packages/pebble-web/src/components/DateInput.tsx:111]: no `inputProps` is of type `Omit<SimpleInputProps, "value" | "onChange" | "placeholder">`

**Review by zhirzh — APPROVED** [review]

**Pull request merged**